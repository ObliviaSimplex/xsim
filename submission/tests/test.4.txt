# =-=-=-=-=-=-=-=-=-=-=-
# Test 4: Bit shifting
# =-=-=-=-=-=-=-=-=-=-=-

This was just a basic test on some bitshifting operations. There were no
surprises here. The expected (and confirmed) output can be found in
test.4.out.

