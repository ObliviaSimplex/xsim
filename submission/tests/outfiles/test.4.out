PC: 040c, State: 0002: Registers:
 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0408
PC: 040e, State: 0002: Registers:
 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406
PC: 0410, State: 0002: Registers:
 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0414, State: 0002: Registers:
 aaaa 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0418, State: 0002: Registers:
 aaaa 5555 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 041c, State: 0002: Registers:
 aaaa 5555 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 041e, State: 0002: Registers:
 5555 5555 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0420, State: 0003: Registers:
 5555 5555 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0424, State: 0003: Registers:
 5555 5555 0001 9999 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0426, State: 0003: Registers:
 5555 5555 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0428, State: 0002: Registers:
 5555 5555 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 042a, State: 0002: Registers:
 aaab 5555 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 042c, State: 0002: Registers:
 aaab aaaa 0001 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 042e, State: 0002: Registers:
 aaab aaaa ffff 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0430, State: 0002: Registers:
 aaab aaaa ffff 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0432, State: 0002: Registers:
 0000 aaaa ffff 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0434, State: 0002: Registers:
 0000 aaaa ffff 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0406 0406
PC: 0436, State: 0002: Registers:
 0000 aaaa ffff 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0408
PC: 0008, State: 0002: Registers:
 0000 aaaa ffff 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 0000 040a
CPU has halted.
